test-suite/Math/NumberTheory/ArithmeticFunctionsTests.hs

```diff
@@ -122,6 +122,32 @@ jordan2Oeis = oeisAssertion "A007434" (jordanA 2)
   , 1728, 1584, 2208, 1536
   ]
 
+-- | congruences 1,2,3,4 from https://en.wikipedia.org/wiki/Ramanujan_tau_function
+ramanujanCongruence1 :: NonZero Natural -> Bool
+ramanujanCongruence1 (NonZero n)
+  | k == 1 = (ramanujan n - sigma 11 n) `mod` (2^11) == 0
+  | k == 3 = (ramanujan n - 1217 * sigma 11 n) `mod` (2^13) == 0
+  | k == 5 = (ramanujan n - 1537 * sigma 11 n) `mod` (2^12) == 0
+  | k == 7 = (ramanujan n - 705 * sigma 11 n) `mod` (2^14) == 0
+  | otherwise = True
+  where k = n `mod` 8
+
+-- | congruences 8,9 from https://en.wikipedia.org/wiki/Ramanujan_tau_function
+ramanujanCongruence2 :: NonZero Natural -> Bool
+ramanujanCongruence2 (NonZero n)
+  | (n `mod` 7) `elem` [0,1,2,4] = m `mod` 7 == 0
+  | otherwise                    = m `mod` 49 == 0
+  where m = ramanujan n - (fromIntegral n) * sigma 9 n :: Integer
+
+-- | ramanujan matches baseline from OEIS.
+ramanujanOeis :: Assertion
+ramanujanOeis = oeisAssertion "A000594" ramanujanA
+  [ 1, -24, 252, -1472, 4830, -6048, -16744, 84480, -113643, -115920
+  , 534612, -370944, -577738, 401856, 1217160, 987136, -6905934, 2727432
+  , 10661420, -7109760, -4219488, -12830688, 18643272, 21288960, -25499225
+  , 13865712, -73279080, 24647168
+  ]
```

Comment:
Let's use a longer baseline, e. g., [first 100 values](https://www.wolframalpha.com/input/?i=RamanujanTau%5BRange%5B100%5D%5D). Due to specific nature of computations it is also worth to write tests, comparing [tau(2^k)](https://www.wolframalpha.com/input/?i=RamanujanTau%5B2%5ERange%5B20%5D%5D) and [tau(3^k)](https://www.wolframalpha.com/input/?i=RamanujanTau%5B3%5ERange%5B20%5D%5D) with baselines.